README.md

```diff
@@ -121,7 +121,7 @@ There's a very helpful group of people in the #graphql channel in [the Elm Slack
 
    ```shell
    npm install --save-dev @dillonkearns/elm-graphql
-   # you can now run it locally with the ./node_modules/.bin/elm-graphql binary,
+   # you can now run it locally using `npx elm-graphql`,
```

Comment:
Executing `./node_modules/.bin/elm-graphql` directly is an anti-pattern, since local installations of `elm` and `elm-graphql` won’t be found then.